app/policies/mpi_policy.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-MviPolicy = Struct.new(:user, :mvi) do
+MPIPolicy = Struct.new(:user, :mvi) do
```

Comment:
Does the :mvi that is getting passed to the struct need to change to :mpi or should it remain :mvi?

Reply:
Yeah the above mvi shouldn't matter. The :mpi authorization in the before action corresponds to the actual MPIPolicy.